projects/indicators/src/modules/chevron/chevron.component.ts

```diff
@@ -48,6 +48,12 @@ export class SkyChevronComponent {
   @Input()
   public disabled = false;
 
+  /**
+   * Indicates whether the chevron button should be displayed as a non-interactable element.
+   */
+  @Input()
+  public displayOnly = false;
```

Comment:
Just out of curiosity, why do we need this input when we already have a `disabled` input? Isn't a disabled chevron button effectively the same thing as a display only chevron button?

Reply:
Good question but not quite. A disabled button is still a button. It has semantic value that it's an interact-able element that is currently disabled. It also comes with native browser CSS for disabled buttons that we'd have to overwrite. Since we only want a visible representation of a chevron that's non-interactive, its more semantically appropriate for this to not be a button.

Reply:
Not a bad suggestions. I've updated this to `iconOnly`. Thanks!